graylog2-web-interface/src/components/users/UsersTokenManagement/TokenManagement.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React, {useMemo, useCallback} from 'react';
+
+import {
+  DEFAULT_LAYOUT, ADDITIONAL_ATTRIBUTES, COLUMNS_ORDER,
+} from 'components/users/UsersTokenManagement/constants';
+import { fetchTokens, keyFn } from 'components/users/UsersTokenManagement/hooks/useTokens';
+import type { Token } from 'components/users/UsersTokenManagement/hooks/useTokens';
+import { PaginatedEntityTable } from 'components/common';
+import TokenActions from 'components/users/UsersTokenManagement/TokenManagementActions';
+
+import CustomColumnRenderers from './ColumnRenderers';
+
+
+const TokenManagement = () => {
+  const tokenAction = useCallback(
+    ({ user_id, id:tokenId, NAME:tokenName}: Token) => (
+      <TokenActions userId={user_id} tokenId={tokenId} tokenName={tokenName} />
+    ),
+    [],
+  );
+  const columnRenderers = useMemo(() => CustomColumnRenderers(), []);
+
+  return (
+    <PaginatedEntityTable<Token> humanName="token management"
```

Comment:
We need to add a container and title for the Token list. Here is an example from the Teams page.
![image](https://github.com/user-attachments/assets/96a741cc-c58d-41a0-9ff4-8da0fc2158a3)

Reply:
Just pushed this change.